code/modules/food_and_drinks/food/foods/sandwiches.dm

```diff
@@ -61,8 +61,8 @@
 	tastes = list("bun" = 4, "tofu" = 4)
 
 /obj/item/reagent_containers/food/snacks/roburger
-	name = "roburger"
-	desc = "The lettuce is the only organic component. Beep."
+	name = "Hamborger"
+	desc = "Looking at this makes your flesh feel like a weakness."
```

Comment:
Two minor nitpicks.
First, you kept roburger as an item name in code, line 63. You'll probably want to change that if you want to get rid of the name. This will, however, require you to check the entire codebase for all appearances of "roburger" and making adjustments.
Second, we generally keep snack names lowercase. There's no reason to capitalize "hamborger".